ScanShop.Mobile/ScanShop.Mobile/Views/ProfilePage.xaml

```diff
@@ -69,10 +69,10 @@
                     <Label Text="Discount:" Style="{StaticResource FrameLabel}" />
                 </Frame>
                 <Label Grid.Row="4" Grid.Column="1" Text="{Binding Path=User.Discount, StringFormat='{0}%'}"
-                       TextColor="#FD5A5A" Style="{StaticResource BlackLabel}"/>
+                        TextColor="#FD5A5A" Style="{StaticResource BlackLabel}"/>
```

Comment:
You can also fix this after removing the tabs in QRScanPage.xaml

Reply:
Fixed.